python/podio_gen/generator_utils.py

```diff
@@ -151,6 +185,9 @@ def __init__(self, name, **kwargs):
     else:
       self.namespace, self.bare_type = _get_namespace_class(self.full_type)
 
+    self.julia_type = get_julia_type(cpp_type=self.bare_type, is_array=self.is_array,
```

Comment:
```suggestion
    self.julia_type = get_julia_type(self.bare_type, is_array=self.is_array,
```
Similarly here. No need to specify the keyword here.

Reply:
Here, as well, since we are specifying for others and not for `self.bare_type`, it will look a little odd.
So either, we keep it as 
```
self.julia_type = get_julia_type(cpp_type=self.bare_type, is_array=self.is_array,
                                     array_type=self.array_type, array_size=self.array_size)
```
Or
```
self.julia_type = get_julia_type(self.bare_type, self.is_array,
                                     self.array_type, self.array_size)
```

Reply:
I think that is OK. In this case the difference between the two is that `cpp_type` is always necessary, whereas all the others are in optional.